cristin-import/src/test/java/no/unit/nva/cristin/lambda/PublicationUpdaterTest.java

```diff
@@ -28,6 +31,29 @@ void shouldReturnUpdatedEventWithNullValuesWhenAllEventValuesAreMissing() {
         assertEquals(publication.getEntityDescription().getReference().getPublicationContext(), emptyEvent());
     }
 
+    @Test
+    void shouldUpdatedEventWithTimeWhenExistingEventIsMissingTimeButNewEventHasTimeMissing() {
```

Comment:
Name is wrong?

Reply:
Yes, thank you, updating to:
`shouldUpdatedEventWithTimeWhenExistingEventIsMissingTimeButNewEventHasTime`